diff --git a/linux/JackLinuxTime.c b/linux/JackLinuxTime.c
index 66449d6e..a5ece765 100644
--- a/linux/JackLinuxTime.c
+++ b/linux/JackLinuxTime.c
@@ -143,11 +143,7 @@ static jack_time_t jack_get_microseconds_from_system (void)
 	jack_time_t jackTime;
 	struct timespec time;
 
-#ifdef CLOCK_MONOTONIC_RAW
-	clock_gettime(CLOCK_MONOTONIC_RAW, &time);
-#else
 	clock_gettime(CLOCK_MONOTONIC, &time);
-#endif
 	jackTime = (jack_time_t) time.tv_sec * 1e6 +
 		(jack_time_t) time.tv_nsec / 1e3;
 	return jackTime;
diff --git a/linux/alsa/alsa_rawmidi.c b/linux/alsa/alsa_rawmidi.c
index d5208374..45d1c3de 100644
--- a/linux/alsa/alsa_rawmidi.c
+++ b/linux/alsa/alsa_rawmidi.c
@@ -865,11 +865,7 @@ void *midi_thread(void *arg)
 			struct timespec ts;
 			ts.tv_sec = 0;
 			ts.tv_nsec = wait_nanosleep;
-#ifdef CLOCK_MONOTONIC_RAW
-			clock_nanosleep(CLOCK_MONOTONIC_RAW, 0, &ts, NULL);
-#else
 			clock_nanosleep(CLOCK_MONOTONIC, 0, &ts, NULL);
-#endif
 		}
 		int res = poll((struct pollfd*)&pfds, npfds, poll_timeout);
 		//debug_log("midi_thread(%s): poll exit: %d", str->name, res);
