diff --git a/src/JackTrip.cpp b/src/JackTrip.cpp
index 5a1070b1..e413ea7e 100644
--- a/src/JackTrip.cpp
+++ b/src/JackTrip.cpp
@@ -1215,10 +1215,12 @@ void JackTrip::stop(const QString& errorMessage)
 
     // Emit the jack stopped signal
     if (!audioErrorMsg.isEmpty()) {
+        std::cerr << audioErrorMsg.toStdString() << std::endl;
         emit signalError(audioErrorMsg);
     } else if (errorMessage.isEmpty()) {
         emit signalProcessesStopped();
     } else {
+        std::cerr << errorMessage.toStdString() << std::endl;
         emit signalError(errorMessage);
     }
 }
